components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabWindowFeature.kt

```diff
@@ -32,7 +33,7 @@ class CustomTabWindowFeature(
     private val activity: Activity,
     private val store: BrowserStore,
     private val sessionId: String,
-    @VisibleForTesting internal val handleError: (Exception) -> Unit
+    @VisibleForTesting internal val handleNewWindowRequestError: (Uri) -> Unit
```

Comment:
nit: Not super convinced about the name. Maybe `onFailedWindowRequest`? Both are not ideal though, as it's not the window request failing. We just failed to launch an intent for the URL and want to try something else. Maybe `onLaunchUrlFallback`?

Reply:
Sure, I'll rename to onLaunchUrlFallback.